### PR TITLE
iOS fixes

### PR DESCRIPTION
**Pull request by zmagyar**
- fixed problems created by renaming the SDK function
- added data layer related functions
- updated documentation

### CODE REVIEW

**Comment by cmackay** [comment]
Thanks! I see the formatting has all been changed. It makes it harder to track actual changes and this project uses 2 spaces for indentation in the javascript files.

**Comment by zmagyar** [comment]
Sorry, my editor auto formatted the files. Shall I fix that on some way or will you during the merge? Also can you implement the android version of these changes?

**Comment by zmagyar** [comment]
Fixed one bug and some of the indentation issues

**Comment by cmackay** [comment]
Hmm, yeah I think it is still messed up. Any of the whitespace or odd indentation changes in the  www/analytics.js file, if you could please undo them I would appreciate it. Thanks!

File: www/analytics.js [hunk 1]
```diff
@@ -22,85 +22,85 @@
 'use strict';
 
 var argscheck = require('cordova/argscheck'),
```
Comment (cmackay) [inline www/analytics.js]: here are some

File: www/analytics.js [hunk 2]
```diff
@@ -22,85 +22,85 @@
...
-    utils     = require('cordova/utils'),
-    exec      = require('cordova/exec'),
-    platform  = require('cordova/platform');
+utils = require('cordova/utils'),
+exec = require('cordova/exec'),
+platform = require('cordova/platform');
 
 var Fields = {
```
Comment (cmackay) [inline www/analytics.js]: whitespace

File: www/analytics.js [hunk 3]
```diff
@@ -22,85 +22,85 @@
...
+TRANSACTION_AFFILIATION: '&ta',
+TRANSACTION_ID: '&ti',
+TRANSACTION_REVENUE: '&tr',
+TRANSACTION_SHIPPING: '&ts',
+TRANSACTION_TAX: '&tt',
+USE_SECURE: 'useSecure',
+VIEWPORT_SIZE: '&vp'
 };
 
 var HitTypes = {
```
Comment (cmackay) [inline www/analytics.js]: more here

File: www/analytics.js [hunk 4]
```diff
@@ -114,213 +114,217 @@ function Analytics() {
 }
 
 Analytics.prototype = {
```
Comment (cmackay) [inline www/analytics.js]: ...

**Pull request closed without merging**